src/engine/server/server.go

```diff
@@ -0,0 +1,25 @@
+package server
+
+import (
+	"cf"
+	"db"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/gorilla/mux"
+	"github.com/tedsuo/ifrit"
+	"github.com/tedsuo/ifrit/http_server"
+)
+
+const (
+	PathScalingActions = "/v1/scaling_actions"
```

Comment:
I am trying to make the API a POST to a resource, not a verb/action.  I am good to change to /v1/scale, 

or maybe POST /v1/apps/{app-id}/scale